tests/HUBActionRegistryTests.m

```diff
@@ -58,7 +58,7 @@ - (void)testRegisteringFactoryAndCreatingAction
     HUBActionFactoryMock * const factory = [[HUBActionFactoryMock alloc] initWithActions:@{actionIdentifier.namePart: action}];
     [self.actionRegistry registerActionFactory:factory forNamespace:actionIdentifier.namespacePart];
     
-    XCTAssertEqual([self.actionRegistry createCustomActionForIdentifier:actionIdentifier], action);
+    XCTAssertEqualObjects([self.actionRegistry createCustomActionForIdentifier:actionIdentifier], action);
```

Comment:
If we are testing instance equality `XCTAssertTrue(a == b)` is far more explicit.